awshelper/config.go

```diff
@@ -357,6 +358,25 @@ func GetAWSPartition(config *AwsSessionConfig, terragruntOptions *options.Terrag
 	return arn.Partition, nil
 }
 
+// GetAWSAccountAlias gets the AWS account Alias of the current session configuration, in case of failure fallback to AWS Account ID.
+func GetAWSAccountAlias(config *AwsSessionConfig, terragruntOptions *options.TerragruntOptions) (string, error) {
+	sess, err := CreateAwsSession(config, terragruntOptions)
+	if err != nil {
+		return "", errors.New(err)
+	}
+
+	aliases, err := iam.New(sess).ListAccountAliases(nil)
+	if err != nil {
+		return "", errors.New(err)
+	}
+
+	if len(aliases.AccountAliases) > 0 {
+		return *aliases.AccountAliases[0], nil
```

Comment:
can `res.AccountAliases[0]` have value `nil`?

Reply:
the underlying type is `[]*string`, so a `nil` value is possible, but according to AWS documentation

>AWS supports only one alias per account https://docs.aws.amazon.com/IAM/latest/APIReference/API_ListAccountAliases.html

a `nil` value would suggest a bug in the AWS API, but I will add an extra check.